test/varity/vcf_to_hgvs_test.clj

```diff
@@ -74,15 +74,15 @@
         "chr6" 33086236 "TA" "T" '("NM_002121:c.776delA") ; cf. rs67523850 (deletion in border of UTR)
 
         ;; Duplication
-        "chr2" 47806842 "T" "TGACT" '("NM_000179:c.4062_4065dupGACT"
-                                      "NM_001281492:c.3672_3675dupGACT"
-                                      "NM_001281493:c.3156_3159dupGACT"
-                                      "NM_001281494:c.3156_3159dupGACT"
-                                      "NM_025133:c.*1276_*1279dupAGTC"
-                                      "NM_001190274:c.*1276_*1279dupAGTC") ; cf. rs55740729 (+)
-        "chr2" 26254257 "G" "GACT" '("NM_000183:c.4_6dupACT"
-                                     "NM_001281512:c.4_6dupACT"
-                                     "NM_001281513:c.-146_-144dupACT") ; cf. rs3839049 (+)
+        "chr2" 47806844 "T" "TGATT" '("NM_000179:c.4068_4071dupGATT"
+                                      "NM_001281492:c.3678_3681dupGATT"
+                                      "NM_001281493:c.3162_3165dupGATT"
+                                      "NM_001281494:c.3162_3165dupGATT"
+                                      "NM_025133:c.*1272_*1275dupTCAA"
+                                      "NM_001190274:c.*1272_*1275dupTCAA") ; cf. rs55740729 (+)
+        "chr2" 26254257 "G" "GACT" '("NM_000183:c.5_7dupCTA"
+                                     "NM_001281512:c.5_7dupCTA"
+                                     "NM_001281513:c.-145_-143dupCTA") ; cf. rs3839049 (+)
```

Comment:
`c.5_7dup` and `c.-145_-143dup` correspond to [rs3839049](https://www.ncbi.nlm.nih.gov/snp/rs3839049#hgvs_tab).